src/stylesheets/core/_layout-grid.scss

```diff
@@ -146,6 +146,10 @@ $namespace-grid: ns('grid');
         }
       }
     }
+    .#{$mq-key}\:#{$namespace-grid}offset-none {
```

Comment:
You'd need to wrap this in a `@include at-media($mq-key) {}` wrapper here to not only output the class, but the proper media query as well.